scalafmt-tests/src/test/resources/scala3/OptionalBraces.stat

```diff
@@ -3609,20 +3609,20 @@ object a:
 object a:
    def f(): Unit =
      List(1, 2, 3).foldLeft(1):
-        case (a, b) => a + b
+          case (a, b) => a + b
 <<< coloneol in fewer braces 1, main > sig
 maxColumn = 80
 indent.main = 4
 ===
 object a:
   def f(): Unit =
     List(1, 2, 3).foldLeft(1):
-      case (a, b) => a + b
+          case (a, b) => a + b
 >>>
 object a:
    def f(): Unit =
        List(1, 2, 3).foldLeft(1):
-          case (a, b) => a + b
+              case (a, b) => a + b
```

Comment:
This looks like way too much indentation. Looks like it's applied 3 times, wouldn't really one be enough? The alternative with {}  will have single indentation:
```
  object a:
    def f(): Unit =
      List(1, 2, 3).foldLeft(1) {
        case (a, b) => a + b
        case (a, b) => a + b
      }
```

Reply:
this example uses 4 as the main indent. so there's only one additional level added, to make way for a potential select continuation after it.

Reply:
We haven't yet agreed on the dotty PR, no? I having doubts about whether we should even merge it. If we don't does this change still make sense?

Reply:
i don't know. it fixes an inconsistency but you're right that both are not yet supported.

but if the dotty change isn't merged, we'll have a problem...

are you opposed to the change martin proposed?

Reply:
I think that change might hard to nail especially with the wierd `1-space` rule.

Reply:
i asked about that space as it seemed unexpected. no documentation mentions that one can be within one space of a significant indentation rather than matching it exactly... does the code not match it exactly?

if he agrees with the half-open interval you're proposing, would that unblock it?

Reply:
> if he agrees with the half-open interval you're proposing, would that unblock it?

I think that would be the most logical yeah.

Reply:
@tgodzik i would like to merge this change anyway. i have another pr ready, which makes this configurable, so if the compiler change is made, this is ready, and if not, there's a currently compatible alternative. wdyt?